<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>makeDraft</class>
 <widget class="QWidget" name="makeDraft">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>622</width>
    <height>453</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>makeDraft</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>300</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>OCR A Extended</family>
    </font>
   </property>
   <property name="text">
    <string>Make Draft</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>50</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>OCR A Extended</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Make Draft</string>
   </property>
  </widget>
  <widget class="QLabel" name="ErrorLabel">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>390</y>
     <width>511</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>OCR A Extended</family>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>120</y>
     <width>331</width>
     <height>131</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>OCR A Extended</family>
    </font>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <property name="fieldGrowthPolicy">
     <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
    </property>
    <item row="1" column="0">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <family>OCR A Extended</family>
       </font>
      </property>
      <property name="text">
       <string>Senders Name</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEdit">
      <property name="font">
       <font>
        <family>OCR A Extended</family>
       </font>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <family>OCR A Extended</family>
       </font>
      </property>
      <property name="text">
       <string>Recievers Name</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="amountLabel">
      <property name="font">
       <font>
        <family>OCR A Extended</family>
       </font>
      </property>
      <property name="text">
       <string>Amount</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="amountLineEdit">
      <property name="font">
       <font>
        <family>OCR A Extended</family>
       </font>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="font">
       <font>
        <family>OCR A Extended</family>
       </font>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>DraftID</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEdit_3"/>
    </item>
   </layout>
  </widget>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>-125</x>
     <y>-59</y>
     <width>931</width>
     <height>641</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/guidesign/BackgroundImage.png);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>300</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <zorder>graphicsView</zorder>
  <zorder>pushButton</zorder>
  <zorder>label_5</zorder>
  <zorder>ErrorLabel</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>pushButton_2</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="finalbankingsystem.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>makeDraft</receiver>
   <slot>makeDraftClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>320</x>
     <y>320</y>
    </hint>
    <hint type="destinationlabel">
     <x>310</x>
     <y>226</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>makeDraft</receiver>
   <slot>changeForm()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>175</x>
     <y>320</y>
    </hint>
    <hint type="destinationlabel">
     <x>310</x>
     <y>226</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>makeDraftClicked()</slot>
  <slot>changeForm()</slot>
 </slots>
</ui>
